### DISCUSSION TITLE
Version 1.40.01

### DISCUSSION THREAD

**Discussion by shred86** [shred86/Ortho4XP#12] [Announcements]
This is the first release of the forked version of Ortho4XP. Version numbering will match the official version but with the addition of a 1.40.XX.

## Features
* Code changes to enable using [PyInstaller](https://pyinstaller.org/en/stable/) to bundle Ortho4XP and its dependencies into a single package.
* Added the ability to set an alternate `custom_overlay_src` directory to resolve an issue for some users. The default X-Plane scenery files are split up between `/X-Plane 12/Global Scenery/X-Plane Global Scenery` and `/X-Plane 12/Global Scenery/X-Plane Demo Areas`. So if you set `custom_overlay_src` to the first directory and try to batch build a bunch of tiles, you might get an error that the .dsf file can't be found if its a location where the .dsf files are located in the second directory.
* Automatically saves the same data (active tile, default provider, default zoom level and base folder) that the power button icon does when you close the application using the operating system close button.
* Saves `custom_dem` and `fill_nodata` to global configuration.
* Added console feedback when using "Erase cached data" options and configuration window buttons (Load Tile Cfg, Write Tile Cfg, etc.).
* Includes Windows Python dependency wheel files for gdal and scikit-fmm.
* Adds a bash script to automate the setup process for those that prefer not to use the packaged version.

## Bug Fixes
* Display asterisk next to each tile zoom level number in the Tiles and configuration window if custom zoom levels have been specified.
* If one-click symlink feature is used, added removal of symlink when "Erase cached data" "Tile (whole)" option is used.
* Fixed zones being saved to tile configuration that were outside of the tile location.
* Fixed minor code typos as they are identified.
* Updated and pin requirements to latest working versions.

## Download
See the [Installation section](https://github.com/shred86/Ortho4XP/wiki/Installation) in the [Wiki](https://github.com/shred86/Ortho4XP/wiki) for the latest download links.

<hr /><em>This discussion was created from the release <a href='https://github.com/shred86/Ortho4XP/releases/tag/v1.40.01'>Version 1.40.01</a>.</em>

**Comment by ColinM9991** [comment]
Well done on getting this out as well as incorporating PyInstaller
